`archivezip` test fails on Windows

## Describe the bug

The `archivezip` test fails on Windows because of a difference in line endings(?).

<details>
<summary>Test failure</summary>

```console
--- FAIL: TestScript/archivezip (0.73s)
    testscript.go:397:
        > [!exec:unzip] skip 'unzip not found in $PATH'
        > mksourcedir
        > chezmoi archive --format=zip --output=archive.zip
        > exec unzip -t archive.zip
        [stdout]
        Archive:  archive.zip
            testing: .create                  OK
            testing: .dir                     OK
            testing: .dir/file                OK
            testing: .dir/subdir              OK
            testing: .dir/subdir/file         OK
            testing: .empty                   OK
            testing: .executable              OK
            testing: .file                    OK
            testing: .private                 OK
            testing: .readonly                OK
            testing: .symlink                 OK
            testing: .template                OK
        No errors detected in compressed data of archive.zip.
        > [!freebsd] cmp stdout golden/archive # FIXME whitespace output of unzip is different on FreeBSD
        --- stdout
        +++ golden/archive

        FAIL: testdata\scripts\archivezip.txt:7: stdout and golden/archive differ
```

</details>

## To reproduce

Run `go test ./...` on Windows.

## Expected behavior

The test should run successfully.

## Output of command with the `--verbose` flag

N/A?

## Output of `chezmoi doctor`

N/A?

## Additional context

N/A

--- a/pkg/cmd/testdata/scripts/archivezip.txt
+++ b/pkg/cmd/testdata/scripts/archivezip.txt
@@ -1,3 +1,4 @@
+[windows] skip 'windows line endings confuse cmp and diff'
 [!exec:unzip] skip 'unzip not found in $PATH'
 
 mksourcedir

--- a/pkg/cmd/testdata/scripts/externalzip.txt
+++ b/pkg/cmd/testdata/scripts/externalzip.txt
@@ -1,3 +1,4 @@
+[windows] skip 'zip may not support the --symlinks option'
 [!exec:zip] skip 'zip not found in $PATH'
 symlink archive/dir/symlink -> file
 exec zip -r --symlinks www/archive.zip archive

